_drafts/SC3-1-1-html-has-lang.md

```diff
@@ -0,0 +1,42 @@
+---
+name: SC3-1-1-html-has-lang
+
+description:
```

Comment:
Needs to be `description: |`

Reply:
This seems correct already.

Reply:
It's missing the `|` pipe character. You need it to indicate multiline text in YAML. I don't think this is valid YAML.